packages/pieces/src/lib/apps/zoom/actions/create-meeting.ts

```diff
@@ -0,0 +1,249 @@
+import { AuthenticationType } from "../../../common/authentication/core/authentication-type";
+import { httpClient } from "../../../common/http/core/http-client";
+import { HttpMethod } from "../../../common/http/core/http-method";
+import { HttpRequest } from "../../../common/http/core/http-request";
+import { createAction } from "../../../framework/action/action";
+import { Property } from "../../../framework/property";
+import { MeetingMessageBody, MeetingResponseBody } from "../common/models";
+import { zoomAuth } from "../common/props";
+
+const defaults = {
+  "agenda": "My Meeting",
+  "default_password": false,
+  "duration": 30,
+  "pre_schedule": false,
+
+  "settings": {
+    "allow_multiple_devices": true,
+    "approval_type": 2,
+    "audio": "telephony",
+    
+    "calendar_type": 1,
+    "close_registration": false,
+
+    "email_notification": true,
+    "host_video": true,
+    "join_before_host": false,
+    "meeting_authentication": true,
+    "mute_upon_entry": false,
+    "participant_video": false,
+    "private_meeting": false,
+    "registrants_confirmation_email": true,
+    "registrants_email_notification": true,
+    "registration_type": 1,
+    "show_share_button": true,
+    "host_save_video_order": true
+  },
+
+  "timezone": "UTC",
+  "type": 2
+}
+
+const action = () => {
+  return createAction({
+    name: 'zoom_create_meeting', // Must be a unique across the piece, this shouldn't be changed.
+    displayName: 'Create Zoom Meeting',
+    description: 'Create a new Zoom Meeting',
+    props: {
+      authentication: zoomAuth,
+      topic: Property.ShortText({
+        displayName: "Meeting's topic",
+        description: "The meeting's topic",
+        required: true,
+      }),
+      agenda: Property.LongText({
+        displayName: 'Agenda',
+        description: "The meeting's agenda",
+        required: false,
+      }),
+      password: Property.ShortText({
```

Comment:
Can we added duration number property, make sure display name contains (In Minutes)